### DISCUSSION TITLE
suppress ResizeObserver loop completed with undelivered notifications.

### DISCUSSION THREAD

**Discussion by dh049** [inokawa/virtua#333] [Q&A] [answered]
is there a way to suppress the know ResizeObserver loop errors? very annoying when working locally...

**Comment by inokawa** [comment]
The errors will be dispatched on window so you can ignore them something like this:
https://github.com/inokawa/virtua/blob/24dd4f354279e4452b077ba91a5a45d9ebebd72b/stories/react/advanced/With%20react-beautiful-dnd.stories.tsx#L68-L73
https://github.com/inokawa/virtua?tab=readme-ov-file#what-is-resizeobserver-loop-completed-with-undelivered-notifications-error

It's a common problem with virtual scroller using ResizeObserver. I hope https://github.com/w3c/csswg-drafts/issues/5488 will be resolved...

**Reply by dh049** [reply]
this is not working for me. any other ideas? i still get the error in spite of  e.stopImmediatePropagation();

**Comment by dh049** [comment]
currently using this as my workaround, which im not proud of:
function onError(evt: ErrorEvent) {
      if (evt.message.includes("ResizeObserver")) {
        evt.stopImmediatePropagation();
        evt.stopPropagation();
        evt.preventDefault();
        const resizeObserverErr = document.getElementById("webpack-dev-server-client-overlay");
        resizeObserverErr && resizeObserverErr.remove();
      }
    }
    window.addEventListener("error", onError);

**Reply by inokawa** [reply, marked as answer]
If you don't want webpack-dev-server to display overlay, you can use `runtimeErrors` option to filter out the errors caused by ResizeObserver.

https://webpack.js.org/configuration/dev-server/#overlay
